packages/core/scripts/gas-rebate/index.js

```diff
@@ -385,11 +442,88 @@ async function getUmaPrice() {
   return priceResponse.uma.usd;
 }
 
+async function getHistoricalGasPrice(startBlock, endBlock) {
+  const etherscanApiKey = process.env.ETHERSCAN_API_KEY;
+  if (!etherscanApiKey) {
+    console.error("Missing ETHERSCAN_API_KEY in your environment, falling back to default gas price");
+    return null;
```

Comment:
does this mean that it will default to null if you don't have an API key? I think this should ideally return to some hardcoded default that's in the script which can then be overridden by an argument. Otherwise, it becomes impossible to run this script unless you have a key.

Reply:
Good idea, will set fallbacks